subsys/bluetooth/controller/Kconfig

```diff
@@ -1001,6 +1004,13 @@ config BT_CTLR_HCI_CODEC_AND_DELAY_INFO
 	  Enable HCI commands to read information about supported
 	  codecs, codec capabilities, and controller delay.
 
+config BT_CTLR_SUBRATING
+	bool "Subrate Connection commands"
```

Comment:
```suggestion
	bool "LE Enhanced Connection Update"
```

Reply:
Is this not what the spec calls it? 
![image](https://github.com/zephyrproject-rtos/zephyr/assets/90382072/23683d85-cada-46d4-b0d3-687cfa96abab)

Reply:
I think it makes sense to use the name from the features list. Subrating is also what it is called everywhere else in the spec.

Reply:
I see. That makes sense, however, since the spec rarely mentions it and does not make this relationship very clear and is inconsistent, I would prefer to keep 'LE Connection Subrating' if you don't have a strong objection. I think this will be more understandable to most people.

Reply:
I have no objection to keep as "LE Connection Subrating"